CellView.pde

```diff
@@ -141,6 +143,9 @@ class CellView extends ViewBase {
                         if (cellModel.codonModels.size() != 0) {
                             float previousCodonAngle = cellModel.codonModels.get(cellModel.previousExecuteHandPosition).segmentAngleInCodonCircle;
```

Comment:
I still get a OutOfBounds exception. Processing doesn't show me the location, but I think it is here. I think you should apply a simular fix for this block.